packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -113,19 +112,24 @@ export default class QuanticSearchBox extends LightningElement {
   }
 
   updateState() {
-    if(this.state.value !== this.searchBox.state.value) {
+    if (this.state.value !== this.searchBox.state.value) {
       this.updateSearchboxText(this.searchBox.state.value);
     }
     this.state = this.searchBox.state;
   }
 
   setSearchBoxContainerClass() {
-    if(this.withoutSubmitButton){
-      this.searchBoxContainerClass = "slds-combobox__form-element slds-input-has-icon slds-input-has-icon_left-right";
-      this.input.setAttribute("aria-labelledby", "fixed-text-label")
-    } else{
-      this.searchBoxContainerClass = "slds-combobox__form-element slds-input-has-icon slds-input-has-icon_right slds-input-has-fixed-addon";
-      this.input.setAttribute("aria-labelledby", "fixed-text-label fixed-text-addon-post");
+    if (this.withoutSubmitButton) {
+      this.searchBoxContainerClass =
+        'slds-combobox__form-element slds-input-has-icon slds-input-has-icon_left-right';
```

Comment:
Can we put this as either top constants at the top of the file? or at least const in the method, not as inline string?

Reply:
Absolutely 👍